main.go

```diff
@@ -156,12 +156,12 @@ func (s *DB) Not(query interface{}, args ...interface{}) *DB {
 }
 
 // Limit specify the number of records to be retrieved
-func (s *DB) Limit(limit int) *DB {
+func (s *DB) Limit(limit interface{}) *DB {
 	return s.clone().search.Limit(limit).db
 }
 
 // Offset specify the number of records to skip before starting to return the records
-func (s *DB) Offset(offset int) *DB {
+func (s *DB) Offset(offset int64) *DB {
```

Comment:
Yes it should

> On Jun 22, 2016, at 8:47 PM, Jinzhu <notifications@github.com> wrote:
> 
> In main.go:
> 
> >  	return s.clone().search.Limit(limit).db
> >  }
> >  
> >  // Offset specify the number of records to skip before starting to return the records
> > -func (s *DB) Offset(offset int) *DB {
> > +func (s *DB) Offset(offset int64) *DB {
> this should be interface, yes?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>